.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+
+<!--
+Thank you for your interest in contributing to XCSoar! Please fill out the
+following questions to make it easier for us to review your changes.
+
+You do not need to check all the boxes below all at once, feel free to take
+your time and add more commits. If you're done and ready for review, please
+check the last box.
+-->
+
+
+What is the purpose of this pull request? What does it change?
+--------------------------------------------------------------
+
+<!--
+Please describe the changes here, as detailed as needed.
+-->
+
+
+Has the changes been discussed in an issue or the forum?
+--------------------------------------------------------
+
+<!--
+Please link issues and relevant forum posts here.
+
+If this PR resolves an issue on GitHub, write "Closes #1234" so that the issue
+is closed automatically when this PR is merged.
+-->
```

Comment:
An ideal PR doesn't have *any* text. People tend to write large stories in PR texts, but have lousy commit messages. PR text gets lost forever in GitHub's proprietary database, but for research (which I do very often to find causes of bugs!), only commit messages stay. Commit messages and code comments are utterly important, and this whole template only distracts from writing good commit messages.
No text at all in the PR, please! If anything, then the template should point that out.

Reply:
I agree all relevant info should be in the commit messages. This part of the template isn't meant to replace that, but just act as a summary of the PR. It's not really clear though, so I'll do something about it.

Reply:
I've reworked most of the PR template, but I did keep these two sections although with some different wording. I especially made the description section clearly be just a brief summary section.